Source/JavaScriptCore/runtime/Options.cpp

```diff
@@ -765,6 +765,13 @@ void Options::notifyOptionsChanged()
     if (thresholdForGlobalLexicalBindingEpoch == 0 || thresholdForGlobalLexicalBindingEpoch == 1)
         Options::thresholdForGlobalLexicalBindingEpoch() = UINT_MAX;
 
+#if !ENABLE(OFFLINE_ASM_ALT_ENTRY)
+    if (Options::useGdbJITInfo())
+        dataLogLn("useGdbJITInfo should be used with OFFLINE_ASM_ALT_ENTRY");
+#endif
+    if (Options::useGdbJITInfo() && Options::useConcurrentJIT())
+        dataLogLn("useGdbJITInfo should be used with JSC_useConcurrentJIT=0");
```

Comment:
I think this does not make sense as code registration happens before starting using it.